compiler/rustc_abi/src/layout.rs

```diff
@@ -119,6 +119,8 @@ impl<Cx: HasDataLayout> LayoutCalculator<Cx> {
             .chain(Niche::from_scalar(dl, Size::ZERO, a))
             .max_by_key(|niche| niche.available(dl));
 
+        let combined_seed = a.size(&self.cx).bytes().wrapping_add(b.size(&self.cx).bytes());
```

Comment:
Why do we need to do any of this seed computation if `-Zrandomize-layout` is not set?

Reply:
Because if `-Zrandomize-layout` is not set, then keeping this set to 0 means we should also be able to avoid all that `//@ normalize-stderr-test:` below.

Reply:
Because randomization is toggled on a per-crate basis, so we want types from non-randomized crates still contribute a seed to randomized crates.

Additionally one builder builds with randomization enabled, which includes std, so it would't actually help the tests.